js/exporter/jspdf/export_data_grid_2.js

```diff
@@ -48,7 +48,28 @@ function exportDataGrid(doc, dataGrid, options) {
                             pdfCell.colSpan = cellMerging.colspan;
                         }
                     } else if(rowType === 'group') {
-                        pdfCell.colSpan = columns.length - 1;
+                        if(cellData.cellSourceData.groupSummaryItems?.length > 0) {
+                            pdfCell._summaryItem = true;
```

Comment:
'_summaryItem' looks strange: there is no 'summary' in 'pdfCell'
Can we rename it? 'isEmpty' ?